plugin.py

```diff
@@ -58,11 +66,50 @@ def platform() -> str:
         return "unknown-linux-gnu"
 
 
+def open_runnables_in_terminus(window: Optional[sublime.Window], runnables: List[Runnable]) -> None:
+    filtered_runnables = [r for r in runnables if r["kind"] == "cargo"]
```

Comment:
This is actually not correct if we allow non-cargo runnables here also.

Reply:
All runnables in `rust-analyzer` are `kind: cargo` so this is a non-issue.

Reply:
So then the condition can be removed I guess?
(I didn't make the original code so I'm not sure what was the reasoning here)

Reply:
Yeah. The reason why `kind` exists is because the rust-analyzer people are planning on suggesting this feature upstream to core LSP. So in that context, it would make sense for clients to differentiate between the different kinds of runnable. However there is currently only one kind in rust-analyzer, so the filter is pointless.